sensorExamples/tempPressure.js

```diff
@@ -1,17 +1,21 @@
 #!/usr/bin/env node
 // Run before: 
+// Black:
 //    sudo chgrp i2c /sys/class/i2c-adapter/i2c-2/new_device
 //    sudo chmod g+w /sys/class/i2c-adapter/i2c-2/new_device
+// AI:
```

Comment:
This needs to be fixed at the image as well. Fine to have some documentation, but add the udev rule to the system.